packages/workbox-routing/src/_types.ts

```diff
@@ -47,10 +48,11 @@ import './_version.js';
  * @callback ~matchCallback
  * @param {Object} context
  * @param {URL} context.url The request's URL.
- * @param {Request} [context.request] The corresponding request,
- *     if available.
+ * @param {boolean} context.sameOrigin The result of comparing `url.origin`
+ * against the current origin.
+ * @param {Request} context.request The corresponding request.
```

Comment:
As per the TypeScript typing (and the code), `request` is always going to be set. I figured I'd fix that while I was here.